frontend/components/Task.tsx

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+function Task(): JSX.Element {
+  const [List, setList] = useState<string[]>([]);
+
+  interface Taskdata {
```

Comment:
No need to add the word 'data', it doesn't add much to our understanding of that interface represents. The naming convention for interfaces, to avoid mixing it up with functional components, is to add `I` before the name. So in this case it would be `ITask`.